__tests__/connection.test.js

```diff
@@ -50,3 +50,40 @@ describe("/api", () => {
 			});
 	});
 });
+
+describe("/api/reviews/:review_id", () => {
+	test("GET - status 200 - Responds with an object containing the information for a review with the given review_id", () => {
+		return request(app)
+			.get("/api/reviews/1")
+			.expect(200)
+			.then((res) => {
+				expect(typeof res.body).toBe("object");
+				expect(res.body.review.length).toBe(1);
+				expect(typeof res.body.review[0].review_id).toBe("number");
+				expect(typeof res.body.review[0].title).toBe("string");
```

Comment:
I would specifically check the review_id as here we want to make sure we are getting back the specific review - we can tell if we are getting the correct one by checking the ID is 1. The other expects we can leave as typeof